wire-ios-utilities/Source/DeveloperFlag.swift

```diff
@@ -87,7 +87,7 @@ public enum DeveloperFlag: String, CaseIterable {
         case .enableMLSSupport:
             return "MLSEnabled"
         case .showCreateMLSGroupToggle:
-            return "CreateMLSGroupEnabled"
+            return nil
```

Comment:
OK so returning nil here means it is still available for developer builds but not for regular builds?

Reply:
@samwyndham  Yes